cpp/examples/parquet/low-level-api/reader-writer.cc

```diff
@@ -27,7 +27,7 @@
  * reference to the API.
  * The file contains all the physical data types supported by Parquet.
  * This example uses the RowGroupWriter API that supports writing RowGroups optimized for
- *memory consumption
+ * memory consumption.
```

Comment:
You may also want to change this line: https://github.com/apache/arrow/blob/master/cpp/examples/parquet/low-level-api/reader-writer2.cc#L30

Reply:
Thanks for your review!